src/syscall/js/js.go

```diff
@@ -318,6 +321,7 @@ func (v Value) Delete(p string) {
 	runtime.KeepAlive(v)
 }
 
+//go:noescape
```

Comment:
Justification: this will affect whether strings with dynamic contents but statically known length to not escape anymore.